src/renderer/pages/Assets/AssetsList/AssetsList.tsx

```diff
@@ -151,6 +152,8 @@ export const AssetsList = () => {
         />
       )}
 
+      <DerivationsAddressModal />
```

Comment:
am I right that it is a temp solution for tests? Maybe add todo comment to move it to onboarding page when it will be merged? 

https://app.clickup.com/t/8692wzuh6

Reply:
I just removed that